spec/models/authenticator/ldap_spec.rb

```diff
@@ -163,6 +164,20 @@ def authenticate
     let(:username) { 'alice' }
     let(:password) { 'secret' }
 
+    context "when using LDAP" do
+      let(:config) { super().merge(:ldap_role => true) }
```

Comment:
@jrafanie They seem to be. I just tried running the spec without them and it fails. :) The `super()` is found throughout the spec.